File: cs/src/TheInternet.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Linq;
using System.Net;

namespace TddExMicrotest
{
    public class GameUrls {
        public const string WORDS = "https://www.wordgamedictionary.com/word-lists/{0}-letter-words/{0}-letter-words.json";
        public const string OTHER = "https://httpstat.us/{0}";
        public const string TURTLES = "https://www.turtles.com/api/{0}/command/{1}";
    }

    public class TheInternet {
        public WebClient _client = new WebClient();

        public string Get(string url, params object[] parameters){
            string urlWithValues = String.Format(url, parameters);
            Console.WriteLine("Downloading from " + urlWithValues);
            return _client.DownloadString(urlWithValues);
        }

    }
}

File: py/src/TheInternet.py
import requests

class GameUrls():
  WORDS = "https://www.wordgamedictionary.com/word-lists/{0}-letter-words/{0}-letter-words.json"
  OTHER = "https://httpstat.us/{0}"
  TURTLES = "https://www.turtles.com/api/{0}/command/{1}"

class TheInternet():
  def get(self, url: str, parameters):
    urlWithValues = url.format(parameters)
    print(f'Downloading from {urlWithValues}')
    return requests.get(urlWithValues).json()
